README.md

```diff
@@ -69,6 +68,25 @@ is:
     service vboxnet enable
     service vboxnet start
 
+Make sure your username is added to the `vboxusers` group to run 
+VirtualBox, on FreeBSD the procedure is:
+
+    (sudo) pw groupmod vboxusers -m <username>
+
+Set the valid ranges for Virtualbox in `/etc/vbox/networks.conf`:
+
+    mkdir -p /etc/vbox
+    vi /etc/vbox/networks.conf
```

Comment:
That's interesting - seems like this was added to virtualbox fairly recently https://www.virtualbox.org/manual/ch06.html#network_hostonly

I created a patch and opened a bug [here](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=261522) to change the location of that file to /usr/local/etc/vbox. So right now this is correct, hopefully it will change in the future.